jane/doc/extensions/_01-stack-allocation/intro.md

```diff
@@ -108,7 +108,7 @@ Part of the contract of `uses_callback` is that it expects `f` not to
 capture its argument: unexpected results could ensue if `f` stored a
 reference to this table somewhere, and it was later used and modified
 after it was added to the global registry. Using `local_`
-annotations allows this constraint to be made explicit and checked at
+annotations allow this constraint to be made explicit and checked at
```

Comment:
I think the original is correct, with the subject of the sentence being the singular "Using".